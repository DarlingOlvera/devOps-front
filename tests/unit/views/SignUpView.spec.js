import { shallowMount } from "@vue/test-utils";
import Signup from '@/views/SignUpView.vue'


jest.mock('vuetify', () => ({
  useDisplay: jest.fn(() => ({ xs: true })),
}));


jest.mock('vue-router',()=>({
  useRouter: jest.fn(()=>({router:'login'}))
}))

describe('Pruebas en SignUpView', () => {

  test('should match snaptshot', () => {
     const wrapper = shallowMount(Signup)

      //expect(wrapper.html()).toMatchSnapshot()
  });
});
