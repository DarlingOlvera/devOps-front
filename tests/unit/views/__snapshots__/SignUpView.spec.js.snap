// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en SignUpView should match snaptshot 1`] = `
<v-container fluid="" class="pa-5 ma-0">
  <v-row class="d-flex justify-center my-10">
    <h1 class="text-h2 text-title">Crear cuenta</h1>
  </v-row>
  <v-row class="d-flex justify-center">
    <v-card class="pa-10 rounded-xl elevation-0" width="100%" max-width="500" height="auto">
      <form-stub as="form" validationschema="[object Object]" validateonmount="false" keepvalues="false" name="signupForm"></form-stub>
    </v-card>
  </v-row>
</v-container>
`;
